src/content/en/fundamentals/security/prevent-mixed-content/fixing-mixed-content.md

```diff
@@ -298,3 +298,9 @@ The downside of using `block-all-mixed-content` is, perhaps obviously, that all
 content is blocked. This is a security improvement, but it means that these 
 resources are no longer available on the page. This might break features and 
 content that your users expect to be available. 
+
+### Alternatives to CSP
+
+If your site is hosted for you by a platform such as Blogger, you may not have access to modify headers & add a CSP. 
+Instead a viable alternative could be to use a website crawler to find issues across your site for you, such as 
+[HTTPSChecker](https://httpschecker.net/how-it-works#httpsChecker){: .external } or [Mixed Content Scan CLI](https://github.com/bramus/mixed-content-scan){: .external }
```

Comment:
Please split lines at 80 characters, so break the line at the start of `[Mixed Content...`